lib/modules/touch.js

```diff
@@ -0,0 +1,50 @@
+import { contentLoaded } from '../utils';
+
+export default (shower) => {
+    contentLoaded(() => {
+        let action = false;
+        let clickable = false;
+
+        document.addEventListener('touchstart', (ev) => {
+            if (ev.touches.length > 1) {
+                action = false;
+                return true;
```

Comment:
Returning anything from `addEventListener` has no effect; it only matters for `onclick` and friends.